### PR TITLE
Use Travis CI RabbitMQ 

### PR DESCRIPTION
**Pull request by lukebakken**
See travis-ci/travis-ci#6302

I have contacted both Erlang Solutions and Travis CI about this. I'm not sure what's up and I can't reproduce the issue using my local Ubuntu Trusty VMs.

### CODE REVIEW

**Comment by vitaly-krugl** [comment]
Oh man, that's a pain :(. By the way, have you come across these?
* https://askubuntu.com/questions/329450/e-some-index-files-failed-to-download-they-have-been-ignored-or-old-ones-used 
* https://ubuntuforums.org/showthread.php?t=1654068

**Comment by lukebakken** [comment]
@vitaly-krugl I did see something similar but I'm not sure we can do anything about it. I'll add those links to  travis-ci/travis-ci#6302

**Comment by vitaly-krugl** [comment]
@lukebakken, would this help: https://www.rabbitmq.com/install-debian.html#bintray-apt-repo ?

File: .travis.yml [hunk 1]
```diff
@@ -2,19 +2,24 @@ language: python
...
-    sources:
-      - sourceline: deb https://packages.erlang-solutions.com/ubuntu trusty contrib
-        key_url: https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc
-    packages:
-      - erlang-nox
+services:
+  - rabbitmq
+
+# TODO:
+# set 'sudo: false' and uncomment once "Hash Sum mismatch" failure in 'apt-get update' is fixed
```
Comment (vitaly-krugl) [inline .travis.yml]: Where is `apt-get update` called?

File: .travis.yml [hunk 2]
```diff
@@ -27,31 +32,31 @@ stages:
 - name: deploy
   if: tag IS present
 
-cache:
-  apt: true
```
[the remark this answers is not in the document]
Comment (vitaly-krugl) [reply .travis.yml]: I tried apt: false and removing the apt line from cache, but it didn't help.

File: .travis.yml (hunk 1 above)
Comment (lukebakken) [reply .travis.yml]: We don't need it now as we're not installing anything with these changes and using the already-installed Erlang and RabbitMQ

**Pull request merged**